<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tempecalibrateClass</class>
 <widget class="QWidget" name="tempecalibrateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tempecalibrate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:1, y1:0.67, x2:1, y2:0, stop:0.204545 rgba(98, 122, 179, 255), stop:1 rgba(91, 188, 241, 255));

color: rgb(255, 255, 255);
border-style: solid;
border-width: 2px;
border-radius:9px;
}

QGroupBox#groupBox {
	border: 2px solid rgb(61, 139, 255);
	border-radius: 8px;	
	background-color: rgb(221, 221, 221);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{background-color: rgba(38, 38, 38, 200)}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.67, x2:1, y2:0, stop:0.204545 rgba(98, 122, 179, 255), stop:1 rgba(91, 188, 241, 255));

color: rgb(255, 255, 255);
border-style: solid;
border-width: 2px;
border-radius:9px;
border-color: rgb(61, 139, 255);</string>
     </property>
     <property name="text">
      <string>温度校准</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_channel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温度路数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edittemp">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>效准值</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_close">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ok">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
